app/web/features/queryKeys.ts

```diff
@@ -15,6 +15,12 @@ export function userKey(userId?: number) {
   return userId === undefined ? "user" : ["user", userId];
 }
 
+export function liteUserKey(userId?: number) {
+  return userId === undefined ? "liteUser" : ["liteUser", userId];
+}
```

Comment:
This is one of those where it's been a while so I don't remember why we did it like this... but upon fresh perspective, it feels weird/unintuitive to allow `userId` to be undefined? 🤔 Or is this perhaps a way to ensure fetching deleted users won't crash the hook?

Reply:
It seems we pass ids from other queries that could be undefined (if the query fails I guess). For example in app/web/features/communities/discussions/DiscussionPage.tsx.

I agree it's kinda silly but on first glance we'd need to do a bunch of refactoring for this change. Probably makes more sense in a separate PR IMO.